RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 # KBase Collections Release Notes
 
+## 0.1.3
+
+* Added BBMap tool to the CDM pipeline.
+* Included metadata file generation after each tool's execution.
+* Updated Python library dependencies to the latest versions.
+* Standardized thread management logic across all tools.
+* Pass `job_id` to the tool container and remove `node_id`.
```

Comment:
👍 But I think we should keep release notes on a very high level. Otherwise we should just auto generate the release note with every PR in between of releases.

Reply:
Release notes should describe any bugfixes or changes in behavior that the user might encounter, and be readable to the user, which the auto generated release notes are not.

what we should do is just keep the release notes up to date with each PR, then we don't have to spelunk through the logs later

A couple other things I see in the logs

* the task generator now asks for confirmation before submitting a job
* fix working dir name prefix
  * not sure if this is visible to the user or not

Reply:
> We've never taken release notes seriously before

I absolutely take release notes seriously.

The reason they're now required is that collections is released in prod

Reply:
The release notes you link are the initial release, so there's no changes to document

Reply:
Anyway, LGTM now
:+1: